modules/programs/jujutsu.nix

```diff
@@ -33,24 +33,6 @@ in {
         for options.
       '';
     };
-
-    enableBashIntegration = mkOption {
-      type = types.bool;
-      default = true;
-      description = "Whether to enable Bash integration.";
-    };
-
-    enableZshIntegration = mkOption {
-      type = types.bool;
-      default = true;
-      description = "Whether to enable Zsh integration.";
-    };
-
-    enableFishIntegration = mkOption {
-      type = types.bool;
-      default = true;
-      description = "Whether to enable Fish integration.";
-    };
```

Comment:
These should have a `mkRemoveModuleOption` to explain why they were removed.

Reply:
Also, please squash your commits.

Reply:
I've applied these suggestions. Thanks!